### PR TITLE
chore: update release workflow

### PR DESCRIPTION
**Pull request by d3lm**
Tiny update to the PR description for the release PR. Noticed there was punctuation missing at the end 🙈.

### CODE REVIEW

**Review by AriPerkkio — CHANGES_REQUESTED** [review]
CLI's PR commit has same issue:

https://github.com/stackblitz/tutorialkit/blob/8b8b1caea8793748d9946e163a184a3ecb958358/.github/workflows/publish-release.yaml#L67-L75

**Comment by d3lm** [comment]
Wait, what do you mean has some issues?

**Comment by AriPerkkio** [comment]
I mean let's apply this same change to CLI's PR comments. It's also missing the dot. 

```diff
      - name: Create Pull Request
        uses: peter-evans/create-pull-request@v6
        with:
          # Note: `publish-release.yaml` checks explicitly for this commit message
          commit-message: 'chore: release tutorialkit CLI, version: ${{ steps.resolve-release-version.outputs.version }}'
          title: 'chore: release tutorialkit CLI, version: ${{ steps.resolve-release-version.outputs.version }}'
-         body: 'Bump tutorialkit CLI to version ${{ steps.resolve-release-version.outputs.version }}'
+         body: 'Bump tutorialkit CLI to version ${{ steps.resolve-release-version.outputs.version }}.'
          reviewers: SamVerschueren,d3lm,Nemikolh,AriPerkkio
          branch: chore/release-cli-${{ steps.resolve-release-version.outputs.version }}
```

**Comment by d3lm** [comment]
Ahhh ok got it! Yep can do, didn't see that. Good point 👍

File: .github/workflows/publish-release.yaml [hunk 1]
```diff
@@ -68,9 +68,9 @@ jobs:
         uses: peter-evans/create-pull-request@v6
         with:
           # Note: `publish-release.yaml` checks explicitly for this commit message
-          commit-message: 'chore: release tutorialkit CLI, version: ${{ steps.resolve-release-version.outputs.version }}'
-          title: 'chore: release tutorialkit CLI, version: ${{ steps.resolve-release-version.outputs.version }}'
-          body: 'Bump tutorialkit CLI to version ${{ steps.resolve-release-version.outputs.version }}'
+          commit-message: 'chore: release TutorialKit CLI v${{ steps.resolve-release-version.outputs.version }}'
```
Comment (AriPerkkio) [inline .github/workflows/publish-release.yaml:71]: Commit message is used here:

https://github.com/stackblitz/tutorialkit/blob/8b8b1caea8793748d9946e163a184a3ecb958358/.github/workflows/publish-release.yaml#L84

Comment (d3lm) [reply .github/workflows/publish-release.yaml:71]: What do you mean? I am confused 😅

Comment (AriPerkkio) [reply .github/workflows/publish-release.yaml:71]: We'll need to change that line 84 to match the new changed commit message as well:

```diff
- if: ${{ contains(github.event.head_commit.message, 'release tutorialkit CLI') }
+ if: ${{ contains(github.event.head_commit.message, 'release TutorialKit CLI') }
```

Comment (d3lm) [reply .github/workflows/publish-release.yaml:71]: Ah got ya! I didn't understand that from your message that I had to change something, sorry. I ll update that as well. That's for the pointer.

**Review by AriPerkkio — APPROVED** [review]
Looks good, thanks!

**Review by Nemikolh — APPROVED** [review]

**Pull request merged**